src/main/resources/db/changelog/changes/v2.1/archiveAuthorityRecord.sql

```diff
@@ -14,7 +14,7 @@ BEGIN
 
   RETURN NEW;
 END;
-$$
+'
```

Comment:
Why this change needed? It will change hashsum of changeset.

Reply:
It is a rollback change which changed the hashsum and that's why I am reverting it here, so it was ```'``` before.